app/models/event.rb

```diff
@@ -28,19 +28,16 @@ def initialize(data)
   end
 
   def find_user
-   # user_paths = {"CreateEvent" => lambda {data.actor.login},
-   #  "PullRequestEvent" => lambda {data.actor.login},
-   #  "PushEvent" => lambda {data.actor.login}
-   # }
-   # user_paths[data.type].call
-     data.actor.login 
+     data.actor.login
   end
 
   def type
+    #add Delete, Issues
     types = {"CreateEvent" => lambda {"Create " + data.payload.ref_type.humanize},
      "PullRequestEvent" => lambda {"Pull Request"},
      "PushEvent" => lambda {"Push"}
     }
+    types.default = lambda {data.type}
```

Comment:
Use the new lambda literal syntax `-> {...}`.<br>Space missing inside {.<br>Space missing inside }.